### ISSUE TITLE
No notifications on new SMS with new release

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#364]
```

What steps will reproduce the problem?
1. Enable GtalkSMS SMS notification
2. Get an SMS

What is the expected output?

I'm notified via gtalk.

What do you see instead?
nothing

I am able to get replies when I type "s", and I'm able to be notified about 
sent and delivered SMS, but not receiving one.

Additional informations:
Phone model: nexus 4, vanir.
```

Original issue reported on code.google.com by `catlvr...@gmail.com` on 16 Aug 2014 at 3:38

**Comment by GoogleCodeExporter** [comment]
```
Hello

Can you give me some logs? I cannot reproduce this issue.
Can you try to reboot your phone (or force stop and restart the app) ? The SMS 
listener might not be correctly registered.

Are you using Hangout ? In that case, do you have the same issue with the 
command "where" ?

Thanks
```

Original comment by `Florent....@gmail.com` on 16 Aug 2014 at 10:19
- Changed state: **NotReproduced**

**Comment by GoogleCodeExporter** [comment]
```
There's nothing relevant in the logs because when the Sms comes in, gtalksms 
doesn't hook into anything as other apps are doing.

I'm not using Hangouts, and the where command works fine. I've rebooted the 
phone, installed and uninstalled the app several times.
```

Original comment by `catlvr...@gmail.com` on 16 Aug 2014 at 5:56

**Comment by GoogleCodeExporter** [comment]
```
When you generate the logs can you confirm that the option is enabled by 
looking for the notificationIncomingSmsType setting.

Did you try the "where" command?
```

Original comment by `Florent....@gmail.com` on 26 Aug 2014 at 1:58

**Comment by GoogleCodeExporter** [comment]
```
Ok for the where command I misread your previous comment.
Can you provide me some logs ?
Or give me at least the result of the command "set:notificationIncomingSmsType"

```

Original comment by `Florent....@gmail.com` on 13 Oct 2014 at 8:06

**Comment by GoogleCodeExporter** [comment]
```
Issue 371 has been merged into this issue.
```

Original comment by `Florent....@gmail.com` on 13 Oct 2014 at 8:07

**Comment by GoogleCodeExporter** [comment]
```
Log sent (from issue 371) -- Notification is enabled.  I don't even get the 
banner message when I connect, FYI.


```

Original comment by `rod...@krweb.net` on 13 Oct 2014 at 8:09

**Comment by GoogleCodeExporter** [comment]
```
By reading the logs everything looks ok.
Can you try to execute command from the app ? Go to the tab "command" and send 
the command "batt" then send me the logs.

Thanks
```

Original comment by `Florent....@gmail.com` on 13 Oct 2014 at 8:13

**Comment by GoogleCodeExporter** [comment]
```
For some reason I couldn't actually "send" the command -- I could enter it, but 
the send button would not take.  I emailed the logs.

I notice looking at the log differences between 4.4.4 and 4.4.5 that 4.4.4 
includes more settings related to notification, like notifiedAddress.

```

Original comment by `rod...@krweb.net` on 13 Oct 2014 at 8:22

**Comment by GoogleCodeExporter** [comment]
```
I have also the same problem but even the notification doesn't work for me with 
version 4.4.5. I restore the previous version 4.4.4 (with titanium backup) and 
everything work well. 
```

Original comment by `Sperenz55` on 15 Oct 2014 at 2:50

**Comment by GoogleCodeExporter** [comment]
```
Everyone is using gmail account for GTalkSMS and notifications ?
```

Original comment by `Florent....@gmail.com` on 15 Oct 2014 at 2:57
- Changed state: **Investigating**

**Comment by GoogleCodeExporter** [comment]
```
I'm using gmail account for send and receive my notifications 
```

Original comment by `Sperenz55` on 15 Oct 2014 at 4:21

**Comment by GoogleCodeExporter** [comment]
```
I report the same issue. My log contains this:
10-15 21:06:21.274 D/gtalksms(23415): 
[xmpp.XmppMultipleRecipientManager@filterHangoutAddresses:99] Looking for 
hangout addresses
10-15 21:06:21.284 I/gtalksms(23415): [MainService@onHandleIntentMessage:306] 
handled action 'com.googlecode.gtalksms.action.SMS_RECEIVED' - state now: 
Connected
10-15 21:06:31.344 I/gtalksms(23415): [XmppManager@send:656] Sending message 
"Reply contact is Xxxxxxx +42..."
10-15 21:06:31.354 D/gtalksms(23415): 
[xmpp.XmppMultipleRecipientManager@getAllowedNotifiedAddresses:81] Message not 
sent to xxxxxx@jabbix.xxx/androidWv2SegNi because resource is blacklisted
10-15 21:06:31.354 D/gtalksms(23415): 
[xmpp.XmppMultipleRecipientManager@filterHangoutAddresses:99] Looking for 
hangout addresses
10-15 21:08:17.107 D/gtalksms(23415): 
[xmpp.ChatPacketListener@processPacket:28] XMPP packet received - sending 
Intent: com.googlecode.gtalksms.action.XMPP.MESSAGE_RECEIVED
10-15 21:08:17.107 I/gtalksms(23415): [MainService@onHandleIntentMessage:234] 
handling action 'com.googlecode.gtalksms.action.XMPP.MESSAGE_RECEIVED' while in 
state Connected
```

Original comment by `hubblebu...@gmail.com` on 18 Oct 2014 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
I'm able to reproduce the issue. I'm working on a fix.
I'll keep you posted.
Thanks for your help.
```

Original comment by `Florent....@gmail.com` on 19 Oct 2014 at 1:05

**Comment by GoogleCodeExporter** [comment]
```
New beta 4.4.5.80 available on market. Should be visible in few minutes/hours.
Please keep me posted if it fixed the issue for you guys.
Thanks
```

Original comment by `Florent....@gmail.com` on 19 Oct 2014 at 2:01
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
```
with v5.0 everything is OK for me
Thank you very much !
```

Original comment by `Sperenz55` on 19 Oct 2014 at 10:18

**Comment by GoogleCodeExporter** [comment]
```
Released on Play Market as 5.0
```

Original comment by `Florent....@gmail.com` on 25 Oct 2014 at 6:23
- Changed state: **Deployed**
